### ISSUE TITLE
Obviate need for dv_rebind_bootstrap_ui.js

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#2176]
https://github.com/IQSS/dataverse/blob/master/src/main/webapp/resources/js/dv_rebind_bootstrap_ui.js is a code smell. It was refactored into its own file in #527 which is great but why do we need to rebind Bootstrap in the first place? Let's see if we can obviate the need for this.

Some avenues to pursue:
- BootFaces (by @TheCoder4eu): "BootsFaces plays nicely with PrimeFaces" according to http://bootsfaces.net/integration/PrimeFaces.jsf and as of this writing uses the [latest version of Bootstrap](https://github.com/TheCoder4eu/BootsFaces-OSP/blob/71385887071bd1fa16f78474b9ff5fe508511782/build.properties#L26). Don't get distracted [playing Tetris](https://www.angularfaces.net/tetris/index.jsf) ([code](https://github.com/stephanrauh/AngularTetrisOnBootsFaces) by @stephanrauh) which is based on a combination of BootFaces and AngularFaces ( https://angularfaces.net ).
- ...

**Comment by TheCoder4eu** [comment]
Hi Philip, 
I'm the BootsFaces original Author, thanks for mentioning.
Nice project! I'll have look at it ASAP.
Please let me know if you need help with the Framework in case you decide to use it in your project!

**Comment by stephanrauh** [comment]
Hi Philip,

more to the point, we went into great lengths to ensure the BootsFaces is compatible to PrimeFaces. If there's any issue, don't hesitate to report it. But I'm sure there nothing to report ;).

BTW, thanks for mentioning AngularTetris. You made my day! Still laughing :)

Regards,
Stephan

**Comment by pdurbin** [comment]
@mheppler since I mentioned BootsFaces today I took a peek at this issue. Sure enough, "BootsFaces plays nicely with PrimeFaces" is still promised and now appears at https://showcase.bootsfaces.net/integration/PrimeFaces.jsf . It might be worth a look.

**Comment by scolapasta** [comment]
BootsFaces is one of the packages I learned about from JavaOne last year (and have mentioned to the team) that I planned on exploring. Definitely worth a look.

**Comment by stephanrauh** [comment]
In the meantime, we've learned a lot why so many developers complain about the incompatibility between PrimeFaces and BootsFaces. BootsFaces is meant to be used as a companion framework of PrimeFaces, but Bootstrap and PrimeFaces use a different CSS measuring model. I did something about it. Have a look at [my blog article](https://www.beyondjava.net/blog/how-to-use-bootsfaces-and-primefaces-in-the-same-project/) explaining the problem and [my GitHub repo solving the problem](https://github.com/stephanrauh/PrimeFacesAlongWithBootsFaces) (at least partially, it's still work in progress).

Enjoy!
Stephan
